code/modules/mob/living/carbon/human/species/unathi.dm

```diff
@@ -14,7 +14,7 @@
 	else, frequently even their own lives. They prefer warmer temperatures than most species and \
 	their native tongue is a heavy hissing laungage called Sinta'Unathi."
 
-	species_traits = list(LIPS)
+	species_traits = list(LIPS, NO_HAIR)
```

Comment:
Undo this

Reply:
I still believe it'd be better to keep this so that Unathi won't have their frills falling out, but with two maintainers against it I'll concede the point.